snapd-qt/notice.cpp

```diff
@@ -60,17 +60,17 @@ static QDateTime convertDateTime (GDateTime *datetime)
 
 QDateTime QSnapdNotice::firstOccurred () const
 {
-    return convertDateTime ((GDateTime*) snapd_notice_get_first_occurred (SNAPD_NOTICE (wrapped_object)));
+    return convertDateTime ((GDateTime*) snapd_notice_get_first_occurred2 (SNAPD_NOTICE (wrapped_object)));
```

Comment:
These casts shouldn't be necessary, right?

Reply:
Mmm... yes, they aren't needed anymore. I added there when the return value was `const GDateTime*`, so they were required, but now they aren't. Fixed.